### DISCUSSION TITLE
MySQL Backup [Jun 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2478] [Discord Threads]
> 🙋‍♂️ **WM31**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi  
#  
> 🙋‍♂️ **FrancyMak**  
#### There are several methods. It depends on what kind of hosting you have and what kind of permissions you have 🙂  
#  
> 🙋‍♂️ **WM31**  
#### I am on self hosting and my SQL is in its own container  
#  
> 🙋‍♂️ **WM31**  
#### Do common my SQL back up method work? And once restored the db, anything to do on Nocodb side to restore the setting  
#  
> 💁 **o1lab**  
#### https://gist.github.com/spalladino/6d981f7b33f6e0afe6bb  
#  
> 💁 **o1lab**  
#### hope u've set NC_DB to your mysql  
#  
> 🙋‍♂️ **WM31**  
#### What is this?  
#  
> 🙋‍♂️ **WM31**  
#### I get this and sadly I left that as default, so I think the backup will work, but not that secure 😩  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/986494815044784169)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
